packages/fxa-content-server/app/scripts/views/settings.js

```diff
@@ -107,6 +108,8 @@ const View = BaseView.extend({
     const account = this.getSignedInAccount();
 
     context.set({
+      ccExpired: !!this._ccExpired,
+      escapedCcExpiredLinkAttrs: `href="/subscriptions" class="alert-link"`,
```

Comment:
I think this needs an explicit trailing `/` - so `/subscriptions/` here. It's a 404 without

Reply:
Thanks.

Tangential, but that seems different to the rest of our routes fwiw. `/signin`, `/signup`, `/settings`, `/sms` etc all work fine. Conscious decision?

Reply:
Hmm, not a conscious decision, now that you mention it. Maybe the better fix here is to let this link stay as-is and tweak the server.

Reply:
In fact: https://github.com/mozilla/fxa/pull/1926